### PR TITLE
Thirdhomework

### PR DESCRIPTION
**Pull request by MutalievUtkirbek**

### CODE REVIEW

**Comment by Morphine-medicine** [comment]
Hey, Utkirbek, merge previous pr please and update this one to have only auth changes, so it will be easier to review, and when you will do that, please notify me, by adding 'ready for review' label to this pr, thanks a lot

**Comment by MutalievUtkirbek** [comment]
Merged the previous PR and updated the current branch

File: controllers/auth.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    throw error
+  }
+}
+
+export const signIn = async (req, res) => {
+  try {
+    const user = await User.findOne({ email: req.body.email })
+    if (!user) {
+      return res.status(401).json({
+        message: 'Invalid email or password'
```
Comment (Morphine-medicine) [inline controllers/auth.js:35]: Here we check onlu email, so probably message should be : 'User does not exists'

**Review by Morphine-medicine — CHANGES_REQUESTED** [review]
Mostly looks good, but I didn't find place were you use isAuth middleware, should be added to endpoints

File: controllers/auth.js (hunk 1 above)
Comment (MutalievUtkirbek) [reply controllers/auth.js:35]: i will correct it

**Comment by MutalievUtkirbek** [comment]
Corrected it

**Comment by MutalievUtkirbek** [comment]
Anton, you got me Wrong I know How to use middleware but I Meant I do not know with which route should I use the middleware and we had an online meeting with BAs,  I asked them about it now Know with which routes I should use.

Sorry for Poor description

**Review by Morphine-medicine — APPROVED** [review]

**Comment by Morphine-medicine** [comment]
O, sorry for that misunderstanding. BTW, you are doing right, asking BAs about it)

**Pull request merged**